src/peer_channels.rs

```diff
@@ -636,13 +666,23 @@ mod tests {
             .map(|addr| addr.node_id)
             .collect::<Vec<_>>();
 
-        let alice_iroh = alice.get_or_try_init_peer_channel().await.unwrap();
         assert_eq!(
             members,
-            vec![alice_iroh.get_node_addr().await.unwrap().node_id]
+            vec![
+                alice
+                    .get_or_try_init_peer_channel()
+                    .await
+                    .unwrap()
```

Comment:
Minor, but i'd return `Result` from tests to get rid of `unwrap()` everywhere, it really eats much screen space :)